src/Juvix/Pipeline/Backend/Michelson.hs

```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LiberalTypeSynonyms #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeFamilyDependencies #-}
+{-# LANGUAGE UndecidableInstances #-}
+module Juvix.Pipeline.Backend.Michelson where
+
+import qualified Data.HashMap.Strict as HM
+import qualified Data.Text as Text
+import qualified Data.Text.IO as T
+import qualified Juvix.Backends.Michelson.Compilation as M
+import qualified Juvix.Backends.Michelson.Parameterisation as Param
+import qualified Juvix.Core.Application as CoreApp
+import qualified Juvix.Core.ErasedAnn as ErasedAnn
+import Juvix.ToCore.FromFrontend as FF
+import qualified Juvix.Core.IR as IR
+import qualified Juvix.Core.IR.TransformExt.OnlyExts as OnlyExts
+import qualified Juvix.Core.IR.Typechecker.Types as TypeChecker
+import Juvix.Core.IR.Types.Base
+import Juvix.Core.IR.Types.Globals
+import Juvix.Core.Parameterisation
```

Comment:
In the styleguide it says that will be only including `Juvix.Library` directly, and the rest in a quantified manner. I have seen lots of code that doesn't work this way, so I am not sure how strict we stick to this.

Reply:
Thanks! I'll follow the styleguide